test/framework/robot.py

```diff
@@ -705,21 +702,21 @@ def test_det_easyconfig_paths_from_pr(self):
         gompi_2015a_txt = '\n'.join([
```

Comment:
for consistency, variable name should now be `gompi_2018b_txt`?

Reply:
good point, fixed in f6ae5c688